app/models/comment.rb

```diff
@@ -1,4 +1,4 @@
 class Comment < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
   belongs_to :post
 end
```

Comment:
-  [ ] [REQUIRED] Kindly ensure that the Comment model has a method that updates the comments counter for a post as stated in the project requirement.